### PR TITLE
add documentation about SSL certificate

### PR DESCRIPTION
**Pull request by chemaper**
#### Reviewers
 @jesuspg  @hmunfru  

 #### Description
Documentation concerning SSL certificates.

This is the same than the PaaS part, but with some changes in paths and format.

 #### Testing
Tested that the document showed in the browser is OK

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

File: doc/installation-guide.rst [hunk 1]
```diff
@@ -192,6 +192,7 @@ It is a a maven application:
 ``       $ mvn assembly:assembly``
 
 -  copy target/distribution/puppwrapper-dist to a desired location
+-  run ./generateselfsigned.sh on puppetwrapper-dist/bin
```
Comment (flopezag) [inline doc/installation-guide.rst]: where is the script generateselfsigned.sh?

Comment (chemaper) [inline doc/installation-guide.rst]: on puppetwrapper-dist/bin

Comment (flopezag) [inline doc/installation-guide.rst]: maybe it is better to change to:

run <on_puppetwrapper_dist>/bin/generateselfsigned.sh?

**Comment by flopezag** [comment]
LGTM

**Pull request merged**